src/Common/src/Interop/Windows/kernel32/Interop.Access.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+		/// <summary>
+        /// <a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/kernel/buffer-descriptions-for-i-o-control-codes">RequiredAccess</a>.
+		/// Indicates the type of access that a caller must request when opening the file object that represents the device (see IRP_MJ_CREATE).
+        /// </summary>
+		
+        public enum Access
```

Comment:
You can make this a `[Flags]` to represent that you can OR the values. The other option (given that `Any` is `0`), is to define `FILE_READWRITE_ACCESS` with a comment noting that it isn't actually defined in Windows, but we're including it for convenience.

Reply:
I have added the `[Flags]` attribute to the enum.